docs/root/start/sandboxes/local_ratelimit.rst

```diff
@@ -41,7 +41,9 @@ Step 2: Test rate limiting of upstream service
 
 The sandbox is configured with ``10000`` port for upstream service.
 
-If a request reaches the rate limit, Envoy will add ``x-local-rate-limit`` header and refuse the connection with a 429 HTTP response code and with the content ``local_rate_limited``.
+Two descriptors with generic_key are set in this example, and the route will match the descriptor with generic_key ``test1``.
+
+If a request reaches the descriptor ``test1's`` rate limit, Envoy will add ``x-local-rate-limit`` header and refuse the connection with a 429 HTTP response code and with the content ``local_rate_limited``.
```

Comment:
If a request reaches the rate limit of descriptor ``test1`` ...

Reply:
thanks ...